app/_landing_pages/gateway/deployment-topologies.yaml

```diff
@@ -8,12 +8,224 @@ rows:
+                D
+                E
+                end
+                style id1 stroke-dasharray:3,rx:10,ry:10
+                style id2 stroke-dasharray:3,rx:10,ry:10
+                style id3 stroke-dasharray:3,rx:10,ry:10
+                style B stroke:none,fill:#0E44A2,color:#fff
+                style B2 stroke:none,fill:#0E44A2,color:#fff
+      - blocks:
+          - type: structured_text
+            config:
+              blocks:
+                - type: text
+                  text: |
+                    In hybrid mode, {{site.base_gateway}} nodes in a cluster are split into two roles: Control Plane
+                    (CP), where configuration is managed, and data
+                    plane (DP), which serves traffic for the proxy.
+
+                    Hybrid mode deployments have the following benefits:
+
+                    * Users can deploy groups of Data Planes in different data centers, geographies, or zones without needing a local clustered database for each DP group.
+                    * The availability of the database does not affect the availability of the Data Planes. If a Control Plane is offline, Data Planes will run using their last known configuration.
+                    * Drastically reduces the amount of traffic to and from the database, since only CP nodes need a direct connection to the database.
+                    * If one of the DP nodes is compromised, an attacker won’t be able to affect other nodes in the {{site.base_gateway}} cluster.
+
+                    **Works on:** {{site.konnect_short_name}}, {{site.base_gateway}}
+          - type: button
+            config:
+              align: left
+              text: Learn more about hybrid mode
+              url: /gateway/hybrid-mode/
+  - header:
+      type: h3
+      text: "Supported Data Plane hosting options"
+    columns: 
+      - blocks:
+        - type: card
+          config:
+            title: Serverless gateways 
+            description: |
+              Learn about serverless gateways hosting for Data Planes.
+              
+            icon: /assets/icons/serverless-gateway.svg
+            cta:
+              url: "/gateway/topology-hosting-options/#serverless-gateways"
+      - blocks: 
+        - type: card
+          config:
+            title: Dedicated Cloud Gateways
+            description: |
+              Learn about Dedicated Cloud Gateways hosting for Data Planes.
+              
+            icon: /assets/icons/dedicated-cloud-gateway.svg
+            cta:
+              url: "/gateway/topology-hosting-options/#dedicated-cloud-gateways"
+      - blocks:
+        - type: card
+          config:
+            title: Self-hosted Data Plane nodes
+            description: |
+              Learn about self-hosting Data Plane nodes.
+              
+            icon: /assets/icons/self-hosted.svg
+            cta:
+              url: "/gateway/topology-hosting-options/#self-hosted"
+    
+  - header:
```

Comment:
Non-blocking: The content seems to run together on this page. Could we have a visual demarkation between sections?

![CleanShot 2025-03-20 at 11 14 47](https://github.com/user-attachments/assets/e0664139-8520-4e06-b8da-dd51c38491bd)